src/client/eppo-client.ts

```diff
@@ -987,6 +995,34 @@ export default class EppoClient {
     validateNotBlank(subjectKey, 'Invalid argument: subjectKey cannot be blank');
     validateNotBlank(flagKey, 'Invalid argument: flagKey cannot be blank');
 
+    // Check for override early
+    const overrideVariation = this.overridesStore?.get(flagKey);
```

Comment:
As we discussed, it will be easier now for malicious users to change the flag values in our customer deployments with the JS SDK. I am going to share an internal document where we looked at evolving our obfuscation with the addition of a salt value generated by the server. This feature could be the catalyst for it - if so we would deploy that with our server-side capabilities framework to SDKs that support the evolution.

Reply:
Yeah this is a good point on obfuscation. I imagine this would need to happen upstream as by the team we get here we'd be operating on the hashed and salted flag key